index.html

```diff
@@ -4269,6 +4331,31 @@ <h2 id="GainNode">
         "[Constructor(&lt;a&gt;BaseAudioContext&lt;/a&gt; context, optional &lt;a&gt;GainOptions&lt;/a&gt; options)]interface GainNode : &lt;a&gt;AudioNode&lt;/a&gt;"
         class="idl">
```

Comment:
apparently when you define it in the `<dl>`, then you don't have to do it in the title. I'll fix.